test/pg_regress.sh

```diff
@@ -22,6 +18,11 @@ TESTS=${TESTS:-}
 IGNORES=${IGNORES:-}
 SKIPS=${SKIPS:-}
 
+if [[ -z "${COMMAND}" ]];  then
```

Comment:
Not sure why we need this. Why not have a default, like before, and then override it from CMake? Mandating that one sets COMMAND prevents easily running and debugging this script.

In fact, it seems this should have worked well by just keeping the `PG_REGRESS` variable and overriding it to `pg_isolation_regress`.

Reply:
this was just insurance i found all the invocations, having pg_regress be the default makes sense